### DISCUSSION TITLE
Automatic Reload when making changes to the OpenAPI yaml

### DISCUSSION THREAD

**Discussion by deanpatel2** [PaloAltoNetworks/docusaurus-openapi-docs#989] [Q&A]
Is there any option to reload the site in response to changes to the OpenAPI yaml (like HMR for the yaml)? Or do you have to make a change, run `yarn docusaurus clean-api-docs all`, and then re-generate them with `yarn docusaurus gen-api-docs all`?

**Comment by raldred** [comment]
I don't think this builds in anything like you could try using `inotifywait`
